packages/settings/src/pharmacies.ts

```diff
@@ -0,0 +1,27 @@
+const ALTO_PHARMACY_ID = 'phr_01G9CM93X1NFP1C9H9K50DPKHX';
```

Comment:
these are same across envs?

Reply:
ha writing the same thing, maybe make it `pharmacies.photon.ts`

Reply:
yup!